### ISSUE TITLE
Formula `vdirsyncer` has not implemented #plist, #service or installed a locatable service file

### ISSUE THREAD

**Issue by angelus2014** [Homebrew/homebrew-core#103975]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
❯ brew gist-logs vdirsyncer
Error: No logs.

❯ brew config
HOMEBREW_VERSION: 3.5.2
ORIGIN: https://github.com/Homebrew/brew
HEAD: fba051f98f609634ae4392d0053ddfc627f8cdb3
Last commit: 5 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 83433e39fd34e2ea950d445f065018ff57852598
Core tap last commit: 6 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_EDITOR: nvim
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_NO_ENV_HINTS: set
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 13.1.6 build 1316
Git: 2.32.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.4-arm64
CLT: 13.4.0.0.1.1651278267
Xcode: N/A
Rosetta 2: false

❯ brew config
HOMEBREW_VERSION: 3.5.2
ORIGIN: https://github.com/Homebrew/brew
HEAD: fba051f98f609634ae4392d0053ddfc627f8cdb3
Last commit: 5 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 83433e39fd34e2ea950d445f065018ff57852598
Core tap last commit: 6 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_EDITOR: nvim
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_NO_ENV_HINTS: set
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 13.1.6 build 1316
Git: 2.32.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.4-arm64
CLT: 13.4.0.0.1.1651278267
Xcode: N/A
Rosetta 2: false

❯ brew doctor
Your system is ready to brew.

```

### Verification

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

After installing vdirsyncer, one sees this message: 'To start vdirsyncer now and restart at login:
  brew services start vdirsyncer'
 
To start the launchctl service.

### What happened (include all command output)?

❯ brew services start vdirsyncer
Error: Formula `vdirsyncer` has not implemented #plist, #service or installed a locatable service file

### What did you expect to happen?

That the service would be started and running automatically.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
brew install vdirsyncer
❯ brew services start vdirsyncer
Error: Formula `vdirsyncer` has not implemented #plist, #service or installed a locatable service file
```

**Comment by branchvincent** [comment]
Fixed by #103983. Please reopen if you still encounter an issue!

**Comment by angelus2014** [comment]
Thank you! It's working fine now:
❯ brew services start vdirsyncer
==> Successfully started `vdirsyncer` (label: homebrew.mxcl.vdirsyncer)


---
With regards,

Angelo Machils
